cmake/modules/unittest.cmake

```diff
@@ -55,6 +55,15 @@ include(${ZEPHYR_BASE}/cmake/kobj.cmake)
 add_dependencies(test_interface ${KOBJ_TYPES_H_TARGET})
 gen_kobj(KOBJ_GEN_DIR)
 
+FILE(TOUCH ${APPLICATION_BINARY_DIR}/zephyr/include/generated/offsets.h)
```

Comment:
Minor comment related to commit message.

This is not done at build time, but at CMake configure time.

Reply:
Fixed, thanks